### ISSUE TITLE
Spelling correction - sutrani- 2.2.29

### ISSUE THREAD

**Issue by gitraghuram** [ashtadhyayi-com/data#55]
Please change
चार्थे द्वंद्वः
to
चार्थे द्व**न्**द्वः

**Comment by hrishikeshrt** [comment]
If this is the correct correction, then the same needs to happen in numerous other places as well,

```
 Search results for: द्वंद्व
२.४.२ द्वंद्वश्च प्राणितूर्यसेनाङ्गानाम्
४.२.६ द्वंद्वाच्छः
४.३.१२५ द्वंद्वाद्वुन् वैरमैथुनिकयोः
५.१.१३३ द्वंद्वमनोज्ञादिभ्यश्च
५.२.१२८ द्वंद्वोपतापगर्ह्यात् प्राणिस्थादिनिः
५.४.१०६ द्वंद्वाच्चुदषहान्तात् समाहारे
१.२.६३ तिष्यपुनर्वस्वोर्नक्षत्रद्वंद्वे बहुवचनस्य द्विवचनं नित्यम्
२.२.२९ चार्थे द्वंद्वः
२.४.६८ तिककितवादिभ्यो द्वंद्वे
२.४.६९ उपकादिभ्योऽन्यतरस्यामद्वंद्वे
४.३.८८ शिशुक्रन्दयमसभद्वंद्वेन्द्रजननादिभ्यश्छः
६.२.३४ राजन्यबहुवचनद्वंद्वेऽन्धकवृष्णिषु
६.२.१४१ देवताद्वंद्वे च 
६.३.२५ आनङ् ऋतो द्वंद्वे
६.३.२६ देवताद्वंद्वे च
७.३.२१ देवताद्वंद्वे च
```

**Comment by hrishikeshrt** [comment]
Solved in #59
